content/pages/index.liquid

```diff
@@ -1,9 +1,286 @@
+          <p>
+            <a href="{{study.link | url}}">{{ study.read_me_for }}</a>
+          </p>
+      </div>
+      {% endfor %}
+    </div>
+  </div>
+</section>
+
+<section class="usa-section bg-accent-warm-light">
+  <div class="grid-container maxw-desktop">
+    <div class="grid-row">
+      <div class="tablet:grid-col">
+        <p class="usa-intro tablet:grid-col-10">
+          Want to talk about what Pages can do for your agency? Questions about compliance? Want to try it out for yourself?
+        </p>
+        <p><a class="usa-button usa-button--big" href="{{ '/pages/contact/' | url }}">Get in touch</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="usa-section margin-y-8">
+  <div class="grid-container maxw-desktop usa-prose">
+    <div class="grid-row">
+      <h2>Kickstart your website with one of our templates.</h2>
+      <p>
+        Our accessible, customizable templates let you spend more time on your content.
+      </p>
+    </div>
+    <div class="grid-row grid-gap margin-bottom-2">
+    {% for example_site in example_sites %}
+      <div class="tablet:grid-col-6 padding-2 text-center">
+        <a class="border border-base-lighter display-block margin-bottom-1" href="{{example_site.link | url }}"><img class="display-block" src="{{example_site.screenshot | url }}" alt="Screenshot of {{example_site.name}}"></a>
+        <a href="{{example_site.link | url}}">{{example_site.name}}</a>
+      </div>
+    {% endfor %}
+    </div>
+    <div class="grid-row">
+      <a class="cg-arrow" href="{{ '/pages/documentation/templates/' | url }}">See all templates</a>
+    </div>
+  </div>
+</section>
+
+<section class="usa-section margin-y-8">
+  <div class="grid-container maxw-desktop usa-prose">
+    <div class="grid-row">
+      <h2>Pages fact sheets</h2>
+      <p>
+        Need more details about Pages? Need help convincing your executive team? The Pages team has developed these resources to help. See more on the <a href="{{ '/pages/documentation/' | url }}">Pages support site</a>.
+      </p>
+    </div>
+    <div class="grid-row grid-gap">
+      {% for sheet in fact_sheets %}
+      <div class="tablet:grid-col-4 bar-top">
+        <div class="padding-2">
+          <a href="{{sheet.pdfs[0].link | url }}" class="display-block margin-bottom-2"><img class="border border-base-lighter display-block" src="{{sheet.thumbnail | url }}" alt="{{sheet.thumbnail_alt}}"></a>
+          <h3>{{sheet.name}}</h3>
+          {% for pdf in sheet.pdfs %}
+          <p>
+            {{pdf.description}}
+          </p>
+          <a class="cg-arrow" href="{{pdf.link | url}}">Download PDF</a>
+          {% endfor %}
+        </div>
+      </div>
+      {% endfor %}
+    </div>
+  </div>
+</section>
+
+
+<script>
+
+  if (location.search === "?b") {
```

Comment:
do we need this part? i thought you migrated it already?

Reply:
Likely it's duplicate. Investigating...

Reply:
Removed.